src/main/java/org/opensearch/security/configuration/ConfigurationRepository.java

```diff
@@ -312,6 +325,58 @@ private void waitForSecurityIndexToBeAtLeastYellow() {
         }
     }
 
+    public void initDefaultConfiguration() {
+        AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> {
+            try {
+                if (!clusterService.state().metadata().hasIndex(securityIndex)) {
+                    initalizeConfigTask.complete(null);
```

Comment:
Shouldn't this be triggered after line 336?

Reply:
It should be